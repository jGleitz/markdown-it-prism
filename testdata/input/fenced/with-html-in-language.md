# Test

This is a fenced code block:

```"><img/onerror="alert('hacked')"src=.><span\class="
public class Foo() {
	public Foo(String bar) {
		System.out.println(bar);
	}
}
```
