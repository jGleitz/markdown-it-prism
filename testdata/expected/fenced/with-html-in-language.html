<h1>Test</h1>
<p>This is a fenced code block:</p>
<pre class="language-&quot;&gt;&lt;img/onerror=&quot;alert('hacked')&quot;src=.&gt;&lt;span\class=&quot;"><code class="language-&quot;&gt;&lt;img/onerror=&quot;alert('hacked')&quot;src=.&gt;&lt;span\class=&quot;">public class Foo() {
	public Foo(String bar) {
		System.out.println(bar);
	}
}
</code></pre>
